Makefile

```diff
@@ -114,6 +119,10 @@ test-all: $(BIN)
 test-all-sans-melange: $(BIN)
 	$(BIN) build @runtest @runtest-js @runtest-coq
 
+test-coverage: $(BIN)
+	- $(BIN) build --instrument-with bisect_ppx --force @runtest
```

Comment:
What does `-` do in front of a command?

Reply:
It ignores the exit code allowing the command to fail.

Reply:
We know that not all the tests will run when coverage is enabled, so we do a best case scenario (hence --force).